### DISCUSSION TITLE
Change the default click action ?

### DISCUSSION THREAD

**Discussion by tonarii** [bozdoz/wp-plugin-leaflet-map#252] [Q&A] [answered]
Hello, how can i change the default click action to not open a popup but do something else when i click on a geojson area on my map?
I want to keep the tooltip on mouseover, but Ii want to display a lightbox image when the click action is triggered.
Can you tell me which file (and part) to edit ?
Thanks

**Comment by bozdoz** [comment, marked as answer]
The geojson features are iterated here: https://github.com/bozdoz/wp-plugin-leaflet-map/blob/master/shortcodes/class.geojson-shortcode.php#L154-L170

**Reply by tonarii** [reply]
Nice, thanks
